lib/worker.js

```diff
@@ -13,6 +13,12 @@ module.exports = async function ({ targets }) {
   targets = await Promise.all(targets.map(async (t) => {
     let fn
     try {
+      if(typeof process !== "undefined"){
```

Comment:
Please follow the linting rules of this project.

Reply:
How can this condition happen?